csrc/scheduler/tools/static_repeat.cpp

```diff
@@ -8,126 +8,40 @@
 
 #include <ir/all_nodes.h>
 #include <ir/utils.h>
+#include <logical_domain_map.h>
 #include <scheduler/tools/static_repeat.h>
 
 namespace nvfuser {
 namespace scheduler_tools {
 
 std::optional<StaticRepeatInfo> getMaybeStaticRepeatInfo(
-    TensorView* maybe_repeat_out) {
-  // The pattern to detect:
-  //
-  // broadcast_out = broadcast(input)
-  // expand_out = expand(broadcast_out)
-  // repeat_out = reshape(expand_out)
-  //
-  // Additionally, since maybe_repeat_out is commonly a fusion
-  // output, it is likely there's a cache tv between expand_out and
-  // repeat_out, so the following pattern should also be detected.
-  //
-  // broadcast_out = broadcast(input)
```

Comment:
These 4 lines were helpful.

Reply:
Not sure which 4 lines, but broadcast is no longer required.